### ISSUE TITLE
Remove or update Symfony Console in composer.json?

### ISSUE THREAD

**Issue by jmichaelward** [wp-cli/wp-cli#4745]
I recall asking @danielbachhuber awhile back as to whether Symfony Console was actually used within WP-CLI. I've been tinkering with version 4 of that package, and realized today that I'm not able to install both WP-CLI and Symfony Console 4 due to WP-CLI's requirement of versions ^2.7 or ^3.0.

If WP-CLI is not actively using this package, then the dependency should be removed from the composer.json file so developers can use whichever version of Symfony Console that they need. Alternately, if WP-CLI is not actively using Symfony Console but has plans to do so in the future, perhaps the version could be updated to include version 4?

**Comment by kevinvhengst** [comment]
@jmichaelward I'm facing the same issue. I'm unable to install WP-CLI using Composer.

**Comment by danielbachhuber** [comment]
Related #4749

@jmichaelward @kevinvhengst Are you installing WP-CLI v1.5.0 or `dev-master`?

**Comment by jmichaelward** [comment]
Ah, sorry about that, @danielbachhuber. To answer your previous question, I had ^1.5 as the constraint for WP-CLI and was attempting to install symfony/console:^4.0. I believe you and I had had a one-off conversation about removing Symfony altogether at one point in the past, but I'd assumed that had been taken care of before it cropped up again last week. I'll keep an eye on #4678. Thanks.

**Comment by dking3876** [comment]
Installation of wp-cli is failing due to symfony/console dependency and incompatablility with 4.0.11 this thread indicates this was solved but im still having issues. can someone assist please

**Comment by schlessera** [comment]
@dking3876 No, this has not been solved yet. The goal is to remove Symfony dependencies entirely from the base WP-CLI framework, which we'll do for WP-CLI v2.0.0. The only hard dependency on Symfony will be in `wp-cli/package-command`, which is not needed when you pull in WP-CLI into a Composer project anyway.

For now you can solve this by pulling in WP-CLI as a Phar through tools like [Phive](https://github.com/phar-io/phive) or [Tooly](https://github.com/tommy-muehle/tooly-composer-script).
